packages/public/@babylonjs/viewer-alpha/rollup.config.mjs

```diff
@@ -0,0 +1,25 @@
+import typescript from "@rollup/plugin-typescript";
+//import terser from "@rollup/plugin-terser";

```

Comment:
We decided not to minify the js in the lib directory since this is intended to be used by projects doing their own bundling, not directly in a web page. I left this here anyway though because we will want to minify the dist bundle later.